pkg/mathinspector.macos.spec

```diff
@@ -1,18 +1,17 @@
-# -*- mode: python -*-
-
-import os
-VERSION = "0.9.1"
+# -*- mode: python ; coding: utf-8 -*-
 
 block_cipher = None
 
-added_files = [
-  ('assets', 'assets')
-]
+try:
+    from mathinspector.verison import VERSION
+except ImportError:
+    VERSION = "UNKNOWN"
 
-a = Analysis(['mathinspector/__main__.py'],
-             pathex=[os.path.join(os.path.abspath('.'), 'mathinspector')],
+a = Analysis(['../mathinspector/_run.py'],
+             pathex=[os.path.abspath('..')],
              binaries=[],
-             hiddenimports=['pywt._extensions._cwt'],
+             datas=[('../mathinspector/assets', 'assets')],
+             hiddenimports=['pywt._extensions._cwt', 'tcl8'],
```

Comment:
I assume `tcl8` works here instead of the old hacky `Makefile::cp -r`

Reply:
Welcome to the pyinstaller nightmare LOL!  The simple answer is I will confirm this works across all platforms and OS types, the long version is that it's not as simple as your guess, the actual answer depends on an extremely large number of factors and many aspects of this are still unclear.  This is also the root cause of the open issue which is causing the app to crash on Mac OS 10.14